documentation/usage/maven/index.md

```diff
@@ -205,8 +205,8 @@ Configuration can also be supplied directly via the command-line using JVM syste
 
 ### Config files
 
-[Config files](/documentation/configfiles) are supported by the Flyway Maven plugin. If you are not familiar with them,
-check out the [Flyway config file structure and settings reference](/documentation/configfiles) first.
+[Config files](/documentation/configuration/configfiles) are supported by the Flyway Maven plugin. If you are not familiar with them,
```

Comment:
All these links to the `configfiles` page should be `configFiles` (might have been originally broken by my commit, but should get fixed anyway)